Resources/Prototypes/Entities/Markers/Spawners/Random/bluespace_harvester.yml

```
@@ -0,0 +1,709 @@
+        state: implanter0
+  - type: RandomSpawner
+    prototypes:
+      - BikeHornImplanter
+      - LightImplanter
+      - MindShieldImplanter
+      - SadTromboneImplanter
+      - TrackingImplanter
+    chance: 1.0
+    offset: 0.0
+    rarePrototypes:
+      - MacroBombImplanter
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)